### PR TITLE
BTS-818: convert graph test

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

This converts the gharial API test to javascript. BTS-827 found meanwhile, remains Todo after this.

- [x] :hankey: Bugfix

### CODE REVIEW

**Review by adityamukho — COMMENTED** [review]
A few suggestions. Otherwise LGTM.

File: tests/js/client/shell/shell-api-general-graph.js [hunk 1]
```diff
@@ -0,0 +1,1451 @@
...
+
+function get_vertex (graph_name, collection, key) {
+  let cmd = vertex_endpoint(graph_name, collection, key);
+  let doc = arango.GET_RAW(cmd);
+  return doc;
+}
+
+function update_vertex (waitForSync, graph_name, collection, key, body, keepNull, options={}) {
+  let cmd = vertex_endpoint(graph_name, collection, key) + `?waitForSync=${waitForSync}`;
+  if (keepNull !== '') {
```
[the remark this answers is not in the document]
Comment (dothebart) [reply tests/js/client/shell/shell-api-general-graph.js:150]: it seems that the else case this is `true` in all places invoked.

[the remark this answers is not in the document]
Comment (adityamukho) [reply tests/js/client/shell/shell-api-general-graph.js:150]: Ok, all good then.

**Review by adityamukho — APPROVED** [review]
LGTM

**Comment by dothebart** [comment]
tests green except unrelated replication 2 error.

**Pull request merged**